.github/workflows/workflow.yaml

```diff
@@ -0,0 +1,101 @@
+name: Test
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
```

Comment:
The other SDKs also include a scheduled run:

  schedule:
    - cron: '0 0 * * *'